griptape/drivers/image_query_model/bedrock_claude_image_query_model_driver.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+from typing import Optional
+from attr import field, define
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryModelDriver
+
+
+@define
+class BedrockClaudeImageQueryModelDriver(BaseImageQueryModelDriver):
+    """
+    Attributes:
+        max_output_tokens: Max output tokens to return.
+    """
+
+    max_output_tokens: Optional[int] = field(default=4096, kw_only=True, metadata={"serializable": True})
```

Comment:
Do we want a default set here?

Reply:
I think we should stay consistent with whatever we decide on the native `AnthropicImageQueryDriver`. So whatever comes out of that conversation I'll update here as well.